handbook/people-ops/hiring/reference_check_questions.md

```diff
@@ -12,11 +12,9 @@ Requested references will typically be a former manager(s) and/or colleague(s) a
 
 ## Reference call
 
-People Ops will reach out to each reference to schedule calls. People Ops will conduct the reference calls with the hiring manager present to address any potential concerns or follow-up questions.
+People Ops will coordinate a calls with references and hiring managers. People Ops will conduct the reference calls with the hiring manager present to address any potential concerns or follow-up questions.
 
-## Template
-
-Call intro: "We already think [candidate name] is great, and are looking forward to working with them, so the call of this is to learn how we can best work with them and make them successful here at Sourcegraph."
+## Reference call questions
```

Comment:
If the reference calls are for the HM finding specific pieces of info, do we still want/need to document these questions?

Reply:
I think reference calls should still follow the above template (and those questions should be run by recruiters), but HMs should have specific questions they plan to ask to address any concerns (and since these will vary, we will just keep the standard template – since those will still be asked)

Reply:
Ok, I would just add a note at the end of that list to indicate that is when HM can jump in with their custom questions.

Reply:
Good point– added!